src/Chapter1.hs

```diff
@@ -636,7 +647,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Integer -> Integer
+firstDigit n
+  | n < 10 = n
+  | otherwise = firstDigit (div n 10)
```

Comment:
I see, that for a negative number like `-19`, this implementation will return the negative number itself instead of the last digit because the first check `n < 10` will always succeed for negative numbers.